### ISSUE TITLE
Disable window scroll but not the overlay content's scroll!!!

### ISSUE THREAD

**Issue by kunalvirk** [realjoshharrison/jquery-disablescroll#14]
Thanks for the plugin.

Is there anyway to disable the window's scroll but at the same time enabling the overlay container's body scroll. Please find the attached!!!
![ss](https://user-images.githubusercontent.com/7521349/34904948-93945b98-f875-11e7-9ffe-6bab37854591.png)

**Comment by realjoshharrison** [comment]
Yes, you can call disablescroll on any scrolling element, not just the window. 

I think you’d need to make an alternative scrolling element for all your main content (use a main content wrapper if you already have one), call disablescroll on that, and make sure the modal is outside of that element. 

E.g. 

    <body>
      <div class=“scrollable”>...everything except modal...</div>
      <div class=“modal”>...</div>
    </body>

And something like this - not tested but you get the idea:

    .scrollable {
      width: 100%;
      height: 100%;
      overflow: auto;
      -webkit-overflow-scrolling: touch; /* momentum scrolling on mobiles */
    }

You would call disablescroll on `.scrollable`. May not be ideal but I think this is the only way to do it.
